graphql-server/migrations/20220616131820_create-reaction-table.go

```diff
@@ -0,0 +1,47 @@
+package migrations
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/oursky/likedao/pkg/config"
+	"github.com/uptrace/bun"
+)
+
+func init() {
+	config := config.LoadConfigFromEnv()
+
+	Migrations.MustRegister(func(ctx context.Context, db *bun.DB) error {
+		values := FormatValues{
+			"schema": config.ServerDatabase.Schema,
+		}
+		err := db.RunInTx(ctx, &sql.TxOptions{}, func(ctx context.Context, tx bun.Tx) error {
+			_, err := tx.Exec(Format(`
+				CREATE TABLE IF NOT EXISTS {{.schema}}.reaction (
+					id TEXT PRIMARY KEY,
+					reaction TEXT NOT NULL,
+					address TEXT NOT NULL,
+					target_id TEXT NOT NULL,
+					target_type TEXT NOT NULL,
```

Comment:
can we add type+id constraints for validations at db schema?
e.g. acceptable types, & for e.g. iscn there should be a pattern / for proposal the id should be int, etc